lib/client-api/src/ClientApi.ts

```diff
@@ -47,6 +47,7 @@ const warningAlternatives = {
   addLoaders: `Instead, use \`export const loaders = [];\` in your \`preview.js\`.`,
 };
 
+// TODO: talk to tom about the removal of this
```

Comment:
@tmeasday let's chat about this, how to remove.

Reply:
We can remove these. However, I am going to refactor this code so it can wait until I do that?